M/mdbtools/build_tarballs.jl

```diff
@@ -0,0 +1,62 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "mdbtools"
+version = v"0.8.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/cyberemissary/mdbtools.git", "b753ff36a0f1d88ae8a300ed6712f4aa2ddb7d08")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd mdbtools/
+autoreconf -if
+apk add glib-dev
```

Comment:
Why do you need this?

Reply:
When I change `./configure` it had failed because it said it need glib. But `make` ended up failing anyway. `make` succeeded after I added the `Glib_jll` dependency. I've pushed a commit removing this line to see if it needs it or not.

Reply:
Okay, you are right, it wasn't needed.